package.json

```diff
@@ -158,11 +162,12 @@
   },
   "private": true,
   "dependencies": {
-    "@bower_components/flexibility": "10up/flexibility#^2.0.1",
-    "@bower_components/js-polyfills": "inexorabletash/polyfill#^0.1.34",
-    "@bower_components/pace": "HubSpot/pace#^1.0.2",
-    "@bower_components/semantic-ui-calendar": "mdehoog/semantic-ui-calendar#^0.0.8",
-    "@bower_components/wysihtml": "Voog/wysihtml#^0.5.5",
+    "@bower_components/flexibility": "github:10up/flexibility#^2.0.1",
+    "@bower_components/js-polyfills": "github:inexorabletash/polyfill#^0.1.34",
+    "@bower_components/pace": "github:HubSpot/pace#^1.0.2",
+    "@bower_components/semantic-ui-calendar": "github:mdehoog/semantic-ui-calendar#^0.0.8",
+    "@bower_components/wysihtml": "github:Voog/wysihtml#^0.5.5",
+    "caniuse-lite": "^1.0.30001047",
```

Comment:
It is not a devDependency, please update in `.lock` file or find the corresponding dep using old version.